psalm-baseline.xml

```diff
@@ -1566,6 +1566,10 @@
     <ArgumentTypeCoercion>
       <code>$classes</code>
     </ArgumentTypeCoercion>
+    <InvalidArrayOffset>
+      <code><![CDATA[$mapping['enumType']]]></code>
+      <code><![CDATA[$mapping['options']]]></code>
+    </InvalidArrayOffset>
```

Comment:
I think these 2 issues will disappear once `JoinColumnData` is migrated as well.